modules/nlu/src/views/style.scss

```diff
@@ -1,7 +1,8 @@
 .workspace {
   padding-top: 51px;
   width: 100%;
-  height: calc(100vh - 51px);
+  height: calc(100vh - 20px);
```

Comment:
I know we are not a css hero and probably someone will redo this allover. But just to keep in mind that those weird spacings should eventually be extracted in global variables. Magic numbers also applies in css if we want the codebase to scale. You're allowed not to extract it though.